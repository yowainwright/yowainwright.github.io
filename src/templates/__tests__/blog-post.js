import React from 'react'
import BlogPost from '../blog-post'

describe('BlogPost component', () => {
  it('renders correctly', () => {
    const rendered = shallow(<BlogPost />)

    expect(rendered).toMatchSnapshot()
  })
})
