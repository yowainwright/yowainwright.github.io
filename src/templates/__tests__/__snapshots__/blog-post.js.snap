// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost component renders correctly 1`] = `
<div
  className="gatsby-container"
>
  <nav
    className="social-nav"
  >
    <ul
      className="social-list"
    >
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://github.com/yowainwright"
        >
          Github
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://twitter.com/yowainwright"
        >
          Twitter
        </a>
      </li>
      <li
        className="social-list__item social-list__item--hidden"
      >
        <a
          className="social-list__link"
          href="https://www.npmjs.com/~yowainwright"
        >
          NPMJS
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://instagram.com/yowainwright"
        >
          Instragram
        </a>
      </li>
      <li
        className="social-list__item social-list__item--hidden"
      >
        <a
          className="social-list__link"
          href="https://medium.com/@yowainwright"
        >
          Medium
        </a>
      </li>
      <li
        className="social-list__item social-list__item--hidden"
      >
        <a
          className="social-list__link"
          href="http://ratherbe.in"
        >
          Tumblr
        </a>
      </li>
      <li
        className="social-list__item social-list__item--hidden"
      >
        <a
          className="social-list__link"
          href="https://www.strava.com/athletes/722335"
        >
          Strava
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://www.linkedin.com/in/jeffrywainwright/"
        >
          LinkedIn
        </a>
      </li>
      <li
        className="social-list__item social-list__item--hidden"
      >
        <a
          className="social-list__link"
          href="https://open.spotify.com/user/127322575?si=6zrJlJX9RcaOk5mp-iJnJg"
        >
          Spotify
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="/rss.xml"
        >
          RSS
        </a>
      </li>
    </ul>
  </nav>
  <nav
    className="site-nav"
    id="site-nav"
    itemType="http://schema.org/SiteNavigationElement"
    role="navigation"
  >
    <div
      className="site-nav__container"
    >
      <Link
        className="site-nav__main-item-link"
        to="/"
      >
        <div
          className="site-nav__main-item"
        >
          <h3
            className="site-nav__main-item-symbol"
          >
            j
          </h3>
          <h2
            className="site-nav__main-item-title"
          >
            Jeffry.in
          </h2>
        </div>
      </Link>
      <ol
        className="site-nav__items"
      >
        <li
          className="site-nav__item site-nav__item--about"
        >
          <Link
            className="site-nav__link site-nav__link--about"
            to="/about/"
          >
            About
          </Link>
        </li>
        <li
          className="site-nav__item site-nav__item--archive"
        >
          <Link
            className="site-nav__link site-nav__link--archive"
            to="/archive/"
          >
            Archive
          </Link>
        </li>
      </ol>
    </div>
  </nav>
  <article>
    <header
      className="post__header"
    >
      <h1
        itemProp="headeline"
      >
        foo
      </h1>
      <time
        className="post__time"
      />
    </header>
    <div
      className="wrapper"
    >
      <div
        className="content"
        dangerouslySetInnerHTML={
          Object {
            "__html": undefined,
          }
        }
      />
    </div>
    <footer
      className="post__footer"
    >
      <h3
        className="post__footer-title"
      >
        foo
      </h3>
      <p
        className="post__meta"
      />
      <nav
        className="share-list share-list--blog-bottom"
      >
        <button
          className="share-list__button"
          onClick={[Function]}
        >
          Copy Link
        </button>
        <a
          className="share-list__link share-list__link--facebook"
          href="https://www.facebook.com/sharer.php?u=https://jeffry.inundefined"
        >
          Facebook
        </a>
        <a
          className="share-list__link share-list__link--twitter"
          href="https://twitter.com/intent/tweet?url=https://jeffry.inundefined&text=foo&via=@yowainwright"
        >
          Twitter
        </a>
        <a
          className="share-list__link share-list__link--reddit"
          href="https://reddit.com/submit?url=https://jeffry.inundefined&title=foo"
        >
          Reddit
        </a>
      </nav>
    </footer>
  </article>
  <footer
    className="site-footer"
    itemType="http://schema.org/WPFooter"
    role="contentinfo"
  >
    <div
      className="site-footer__wrapper"
    >
      <div
        className="site-footer__col"
      >
        <p
          className="site-footer__body"
        >
          My online thoughts. I'm 
          <Link
            to="/about"
          >
            <strong>
              Jeffry Wainwright
            </strong>
          </Link>
          â€”a software engineer, artist, runner, and surfer living in Los Angeles. 
          <strong>
            <a
              href="mailto:yowainwright@gmail.com"
            >
              Email me
            </a>
          </strong>
           or reach out on 
          <a
            href="https://github.com/yowainwright"
          >
            <strong>
              Github
            </strong>
          </a>
          , 
          <a
            href="https://instagram.com/yowainwright"
          >
            <strong>
              Instagram
            </strong>
          </a>
          , 
          <a
            href="https://twitter.com/yowainwright"
          >
            <strong>
              Twitter
            </strong>
          </a>
          , 
          <a
            href="https://www.strava.com/athletes/722335"
          >
            <strong>
              Strava
            </strong>
          </a>
          , or 
          <a
            href="https://www.linkedin.com/in/jeffrywainwright/"
          >
            <strong>
              LinkedIn
            </strong>
          </a>
          .
        </p>
      </div>
      <div
        className="site-footer__col"
      >
        <nav
          className="social-footer"
        >
          <ul
            className="social-list"
          >
            <li
              className="social-list__item social-list__item--showing"
            >
              <a
                className="social-list__link"
                href="https://github.com/yowainwright"
              >
                Github
              </a>
            </li>
            <li
              className="social-list__item social-list__item--showing"
            >
              <a
                className="social-list__link"
                href="https://twitter.com/yowainwright"
              >
                Twitter
              </a>
            </li>
            <li
              className="social-list__item social-list__item--hidden"
            >
              <a
                className="social-list__link"
                href="https://www.npmjs.com/~yowainwright"
              >
                NPMJS
              </a>
            </li>
            <li
              className="social-list__item social-list__item--showing"
            >
              <a
                className="social-list__link"
                href="https://instagram.com/yowainwright"
              >
                Instragram
              </a>
            </li>
            <li
              className="social-list__item social-list__item--hidden"
            >
              <a
                className="social-list__link"
                href="https://medium.com/@yowainwright"
              >
                Medium
              </a>
            </li>
            <li
              className="social-list__item social-list__item--hidden"
            >
              <a
                className="social-list__link"
                href="http://ratherbe.in"
              >
                Tumblr
              </a>
            </li>
            <li
              className="social-list__item social-list__item--hidden"
            >
              <a
                className="social-list__link"
                href="https://www.strava.com/athletes/722335"
              >
                Strava
              </a>
            </li>
            <li
              className="social-list__item social-list__item--showing"
            >
              <a
                className="social-list__link"
                href="https://www.linkedin.com/in/jeffrywainwright/"
              >
                LinkedIn
              </a>
            </li>
            <li
              className="social-list__item social-list__item--hidden"
            >
              <a
                className="social-list__link"
                href="https://open.spotify.com/user/127322575?si=6zrJlJX9RcaOk5mp-iJnJg"
              >
                Spotify
              </a>
            </li>
            <li
              className="social-list__item social-list__item--showing"
            >
              <a
                className="social-list__link"
                href="/rss.xml"
              >
                RSS
              </a>
            </li>
          </ul>
        </nav>
      </div>
      <div
        className="site-footer__col"
      >
        <figure
          className="figure--center"
        >
          <img
            alt="Me smiling with a beard"
            className="media--circular"
            src="https://yowainwright.imgix.net/portraits/me-smiling.jpg?w=100&h=100&fit=crop&auto=format"
          />
        </figure>
        <p
          className="site-footer__content"
        >
          <Link
            className="site-footer__link"
            to="/"
          >
            jeffry.in 
            2020
          </Link>
        </p>
      </div>
    </div>
  </footer>
</div>
`;
